src/bun.js/webcore/response.zig

```diff
@@ -2291,6 +2324,15 @@ pub const Fetch = struct {
 
                         if (options.get(ctx, "tls")) |tls| {
                             if (!tls.isEmptyOrUndefinedOrNull() and tls.isObject()) {
+                                if (ssl_config) |conf| {
+                                    conf.deinit();
```

Comment:
is there anycase that this would not be null?

Reply:
nope, you're right. I'll remove this 👍